core/src/main/java/org/apache/accumulo/core/file/FileOperations.java

```diff
@@ -42,10 +42,18 @@
   private static final HashSet<String> validExtensions =
       new HashSet<>(Arrays.asList(Constants.MAPFILE_EXTENSION, RFile.EXTENSION));
 
+  // Sometimes we want to know what files accumulo creates
+  private static final HashSet<String> workingFiles =
+      new HashSet<>(Arrays.asList(Constants.BULK_LOAD_MAPPING, Constants.BULK_RENAME_FILE));
+
   public static Set<String> getValidExtensions() {
     return validExtensions;
   }
 
+  public static Set<String> getWorkingFiles() {
```

Comment:
Since this method is not in a class that only deals with bulk import a more specific name like `getBulkWorkingFiles()` would be better.

Reply:
Yes.  I'm not sure if FileOperations is the best place but getValidExtensions was there.

Reply:
Yeah it feels like it should go elsewhere, but I am not sure where.  I think its ok where it is if it has a more specific name.

Reply:
You could put it in BulkImport since its only used by the bulk import classes.